### PR TITLE
feat: Update UI && top-view z-axis

### PR DESCRIPTION
**Pull request by Glenfiddish**

### CODE REVIEW

**Comment by Glenfiddish** [comment]
image
UI Updated

File: packages/lb-components/src/components/pointCloudView/PointCloudInfos.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ */
+export const SizeInfoForView = ({ perspectiveView }: { perspectiveView: EPerspectiveView }) => {
+  const { pointCloudBoxList, selectedID } = React.useContext(PointCloudContext);
+  const box = pointCloudBoxList.find((i) => i.id === selectedID);
+
+  if (selectedID && box) {
+    const infos =
+      perspectiveView === EPerspectiveView.Back
+        ? [
+            { label: '宽', value: box.width },
```
Comment (Kerwin-L) [inline packages/lb-components/src/components/pointCloudView/PointCloudInfos.tsx:27]: The place needs to be internationalized

File: packages/lb-components/src/components/pointCloudView/PointCloudInfos.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+export const BoxInfos = () => {
+  const ptCtx = React.useContext(PointCloudContext);
+  const { selectedID, pointCloudBoxList } = ptCtx;
+  const box = pointCloudBoxList.find((i) => i.id === selectedID);
+
+  if (selectedID && box) {
+    const { width, depth, height, rotation } = box;
+    const infos = [
+      {
+        label: '长',
```
Comment (Kerwin-L) [inline packages/lb-components/src/components/pointCloudView/PointCloudInfos.tsx:60]: The place needs to be internationalized

File: packages/lb-components/src/index.scss [hunk 3]
```diff
@@ -19,13 +19,17 @@ $prefix: bee;
...
+  }
+
   &__side {
-    max-width: 440px;
     background: transparent;
     overflow: hidden;
     position: relative;
     z-index: 10;
     border-top: 1px solid #e2e2e2;
+    width: 15%;
```
Comment (Kerwin-L) [inline packages/lb-components/src/index.scss:32]: Are other tools displayed properly?

**Pull request merged**